src/skmultiflow/lazy/knn_adwin.py

```diff
@@ -45,30 +45,26 @@ class KNNADWINClassifier(KNNClassifier):
         which determines from which point the algorithm will switch for a 
         brute-force approach. The bigger this number the faster the tree 
         construction time, but the slower the query time will be.
-        
-    nominal_attributes: numpy.ndarray (optional, default=None)
-        List of Nominal attributes. If emtpy, then assume that all attributes are numerical.
-        
-    Raises
-    ------
-    NotImplementedError: A few of the functions described here are not 
-    implemented since they have no application in this context.
-    
-    ValueError: A ValueError is raised if the predict function is called 
-    before at least k samples have been analyzed by the algorithm.
+
+    metric: string or sklearn.DistanceMetric object
+        sklearn.KDTree parameter. The distance metric to use for the KDTree.
+        Default=’euclidean’. KNNClassifier.valid_metrics() gives a list of
+        the metrics which are valid for KDTree.
+
+    Notes
+    -----
+    This estimator is not optimal for a mixture of categorical and numerical
```

Comment:
Is there a workaround for this case? Should we explicit that categoric inputs are going to be treated as numeric features?

Reply:
Good idea. Updated in 31687ca4743f3fa03f752ed182135cbef9fc7901